src/Module/Filer/RemoveTag.php

```diff
@@ -53,28 +91,27 @@ protected function content(array $request = []): string
 			$type = Post\Category::FILE;
 		}
 
-		$logger->info('Filer - Remove Tag', [
+		$this->logger->info('Filer - Remove Tag', [
 			'term' => $term,
 			'item' => $item_id,
 			'type' => $type
 		]);
 
-		if ($item_id && strlen($term)) {
-			$item = Post::selectFirst(['uri-id'], ['id' => $item_id]);
-			if (!DBA::isResult($item)) {
-				return '';
-			}
-			if (!Post\Category::deleteFileByURIId($item['uri-id'], local_user(), $type, $term)) {
-				notice(DI::l10n()->t('Item was not removed'));
-			}
-		} else {
-			notice(DI::l10n()->t('Item was not deleted'));
+		if (!$item_id || !strlen($term)) {
```

Comment:
```suggestion
		if (empty($item_id) || !strlen($term)) {
```
Seems more safe, but it's up to you :-)

Reply:
`$item_id` is a variable set in the same function to the result of a null-coalescing operation, so we don't need the extra `isset()` that `empty()` provides.